test/e2e/basic_test.go

```diff
@@ -116,33 +125,63 @@ func (data *TestData) testDeletePod(t *testing.T, podName string, nodeName strin
 	podIPs := podInterfaces[0].IPs
 	t.Logf("Host interface name for Pod is '%s'", ifName)
 
-	doesInterfaceExist := func() bool {
-		cmd := []string{"ip", "link", "show", ifName}
-		stdout, stderr, err := data.runCommandFromPod(antreaNamespace, antreaPodName, agentContainerName, cmd)
-		if err != nil {
-			if strings.Contains(stderr, "does not exist") {
+	var doesInterfaceExist, doesOVSPortExist func() bool
+	var doesIPAllocationExist func(podIP string) bool
+	if isWindowsNode {
+		doesInterfaceExist = func() bool {
+			cmd := fmt.Sprintf("powershell 'Get-HnsEndpoint | Where-Object Name -EQ %s | Select-Object ID | Format-Table -HideTableHeaders'", ifName)
+			_, stdOut, stdErr, err := RunCommandOnNode(nodeName, cmd)
+			if err != nil {
+				t.Fatalf("Error when querying HNSEndpoint with name %s: %s", ifName, err.Error())
+			}
+			if stdErr != "" {
+				t.Errorf("Error when querying HNSEndpoint with name %s: %s", ifName, stdErr)
```

Comment:
Can stderr and err be handled together like the linux version? Not sure why one case we fatal while another case we error and return false.

Reply:
I thought the stdErr only means the error for this check, but it might not block others. But if handling them together is better, I could change it.